### PR TITLE
build.rb: install pip directly to fix errors

### LINKED ISSUE #153: errors encountered during mkdocs related install
```
[/dev/stdin] +++ Execute: run pip -q install mkdocs mkdocs-bootswatch
[/dev/stdin] ------ BEGIN OUTPUT ------
Compiling /tmp/pip-build-1v7jJr/Jinja2/jinja2/asyncfilters.py ...
  File "/tmp/pip-build-1v7jJr/Jinja2/jinja2/asyncfilters.py", line 7
    async def auto_to_seq(value):
            ^
SyntaxError: invalid syntax

Compiling /tmp/pip-build-1v7jJr/Jinja2/jinja2/asyncsupport.py ...
  File "/tmp/pip-build-1v7jJr/Jinja2/jinja2/asyncsupport.py", line 22
    async def concat_async(async_gen):
            ^
SyntaxError: invalid syntax
```

### PR DESCRIPTION
**Pull request by unclejack**
Fixes #153

### CODE REVIEW

**Comment by erikh** [comment]
Can one of the admins verify this patch?

**Comment by erikh** [comment]
ok to test

**Comment by erikh** [comment]
Thanks!

**Comment by erikh** [comment]
I need to look into the Jenkins system, this patch will have to wait until I can do this.

**Comment by unclejack** [comment]
Please take your time. I just wanted to fix that error.

**Comment by erikh** [comment]
rerun tests

**Comment by erikh** [comment]
rerun tests

(you all may want to unwatch this one)

**Comment by erikh** [comment]
rerun tests

**Comment by erikh** [comment]
rerun tests

**Comment by erikh** [comment]
rerun tests

**Comment by erikh** [comment]
rerun tests

**Comment by erikh** [comment]
rerun tests

**Comment by erikh** [comment]
disabling tests until I can fix jenkins, sorry this'll be a day or two

**Comment by erikh** [comment]
rerun tests

**Comment by erikh** [comment]
rerun tests

**Comment by erikh** [comment]
rerun tests

(sorry!)

**Comment by erikh** [comment]
Ok finally!

I have some review comments.

File: build.rb [hunk 1]
```diff
@@ -38,6 +38,8 @@
 
   copy "dind", "/dind"
 
+  run "wget https://bootstrap.pypa.io/get-pip.py && python2.7 /get-pip.py && rm /get-pip.py"
```
Comment (erikh) [inline build.rb]: can we use `curl | python` here instead? the output is loud and we use curl everywhere else. Then, there's no need for the temporary file either.

**Comment by erikh** [comment]
rerun tests

**Pull request merged**